iree/turbine/kernel/wave/wave.py

```diff
@@ -677,7 +677,7 @@ def test_execute(self, args, kwargs):
                 cache_manager.store_kernel(
                     compiled_wave_vmfb,
                     kernel_usages,
-                    mb.module_op.get_asm(),
```

Comment:
this one would still need to be `mb.module_op.get_asm()` I reckon, since at this stage we are storing into the cache